src/Button/ZoomToExtentButton/ZoomToExtentButton.tsx

```diff
@@ -93,6 +110,9 @@ class ZoomToExtentButton extends React.Component<ZoomToExtentButtonProps> {
     if (!view) { // no view, no zooming
       return;
     }
+    if (!extent && (!center && !zoom)) {
+      logger.warn('Provide either an extent or a center and a zoom.');
```

Comment:
Can be changed to an error message, as the button will not work. And its useful to see which component the message comes from, e.g.
```
logger.error('zoomToExtentButton: You need to provide either an extent or a center and a zoom.');
```